### PR TITLE
fix issue: AttributeError: 'module' object has no attribute 'ie'

### PR DESCRIPTION
**Pull request by MungoWang**
To fix error: AttributeError: 'module' object has no attribute 'ie' when using --driver IE

### CODE REVIEW

**Comment by davehunt** [comment]
I've been waiting for this! Someone using pytest-selenium to test on Windows with Internet Explorer or Edge! :)

File: pytest_selenium/drivers/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import internet_explorer as ie
```
Comment (davehunt) [inline pytest_selenium/drivers/__init__.py]: Can you fix this so flake8 doesn't report an unused warning?

Comment (MungoWang) [inline pytest_selenium/drivers/__init__.py]: Sorry that, I didn't realize the problem, fixed now.

**Comment by davehunt** [comment]
Thanks! I've merged, and tagged v1.5.1 so there should be a new release available soon.

**Pull request merged**